tools/bundle_adjust_tracks.cxx

```diff
@@ -1018,6 +1021,22 @@ static int maptk_main(int argc, char const* argv[])
   //
   // Write the output KRTD files
   //
+  if (config->get_value<bool>("krtd_clean_up") )
+  {
+    std::cout << "Cleaning " << config->get_value<std::string>("output_krtd_dir") << " before writing new files." << std::endl;
+
+    std::ifstream framelist(config->get_value<std::string>("image_list_file"));
```

Comment:
This is not doing exactly what is promised by the description.  It is only deleting the KRTD files that have corresponding image files in the image list.  If the image list is changed you can still end up with extra KRTD files that are not deleted.  I think it would be better to just traverse all files in this directory and delete any files ending in ".krtd".